sdk/appcenter-storage/src/main/java/com/microsoft/appcenter/storage/models/BaseOptions.java

```diff
@@ -45,11 +45,10 @@ public int getDeviceTimeToLive() {
     }
 
     /**
-     * @param lastModified timestamp of when the document was last modified
-     * @return whether a document with the specified TTL has expired since it was last modified
+     * @param expiredAt timestamp of when the document was expired.
+     * @return whether a document has been expired.
```

Comment:
wording "whether a document is expired."

Reply:
fixed~